src/pages/VolunteersJobListingFS.page

```diff
@@ -148,9 +151,15 @@
             <apex:outputField value="{!job.Location_Information__c}" />
             </p>
         </apex:outputPanel>
+        
         <apex:outputPanel id="panelJobOnly" rendered="{!job.Number_of_Shifts__c == 0 || ShowShifts == false}" >
-            <input type="button" value="{!$Label.labelButtonSignUp}"  class="cssJobShiftSignup btn" onclick="OpenSignUpDlg(event.clientX, event.clientY, '{!job.Id}', '', '{!JSENCODE(job.Name)}', ''); return false;"/>
-        </apex:outputPanel>         
+            <input type="button" value="{!$Label.labelButtonSignUp}"  class="cssJobShiftSignup btn" onclick="OpenSignUpDlg(event.clientX, event.clientY, '{!job.Id}', '', '{!JSENCODE(job.Name)}', ''); return false;" style="{!IF(job.External_Signup_Url__c == null, '', 'display: none;')}"/>
```

Comment:
I made the change, as you requested, and committed the file - please let me know if you notice anything else. Thank you.